setup.cfg

```diff
@@ -78,6 +78,9 @@ ignore_missing_imports = True
 [mypy-dbt.adapters.*]
 ignore_missing_imports = True
 
+[mypy-slack_sdk.*]
```

Comment:
I think we were missing this? I get a mypy error without it.

Reply:
@vchan

Reply:
Oh I see you made it a dependency of `make install-dev` but not `dev` in setup.py.

Reply:
Yes, it should be an optional dependency.